platforms/android/tangram/src/main/cpp/androidPlatform.cpp

```diff
@@ -313,26 +318,28 @@ UrlRequestHandle AndroidPlatform::startUrlRequest(Url _url, UrlCallback _callbac
         m_callbacks[requestHandle] = _callback;
     }
 
-    jlong jRequestHandle = static_cast<jlong>(requestHandle);
+    m_jniWorker.enqueue([=](JNIEnv *jniEnv) {
+        jlong jRequestHandle = static_cast<jlong>(requestHandle);
 
-    // Check that it's safe to convert the UrlRequestHandle to a jlong and back.
-    assert(requestHandle == static_cast<UrlRequestHandle>(jRequestHandle));
+        // Check that it's safe to convert the UrlRequestHandle to a jlong and back... cmon :P
+        assert(requestHandle == static_cast<UrlRequestHandle>(jRequestHandle));
```

Comment:
I am fine removing this assert!

Reply:
I kind of like this here just as a reminder to the programmer, I don't think it does any harm.

Reply:
I think we can make it a static_assert. jlong is defined as 64bit currently and uint64_t should also stick to 64bit for a while.

Reply:
And yeah it safes us in case we should change our UrlRequestHandle definition